vespa-osgi-testrunner/src/main/java/com/yahoo/vespa/testrunner/TestReport.java

```diff
@@ -40,7 +40,7 @@ public List<LogRecord> logLines() {
     }
 
     public TestRunner.Status status() {
-        return failedCount > 0 ? FAILURE : inconclusiveCount > 0 ? INCONCLUSIVE : successCount > 0 ? SUCCESS : NO_TESTS;
+        return failures.size() > 0 || failedCount > 0 ? FAILURE : inconclusiveCount > 0 ? INCONCLUSIVE : successCount > 0 ? SUCCESS : NO_TESTS;
```

Comment:
Consider adding parenthesis. Operator precedence might not be obvious for everyone.

Reply:
```suggestion
        return (failures.size() > 0 || failedCount > 0) ? FAILURE : inconclusiveCount > 0 ? INCONCLUSIVE : successCount > 0 ? SUCCESS : NO_TESTS;
```